modulefiles/wflow_wcoss2

```diff
@@ -0,0 +1,14 @@
+#%Module
+
+proc ModulesHelp { } {
+   put stderr "This module loads python environement for running SRW on"
+   put stderr "the NOAA operational machine WCOSS2 (Cactus/Dogwood)"
+}
+
+module-whatis "Loads libraries needed for running SRW on WCOSS2 (Cactus/Dogwood)"
+
+module load intel/19.1.3.304
+module load python/3.8.6
+
+module use /apps/ops/test/nco/modulefiles
+module load core/rocoto/1.3.4
```

Comment:
Do we need to put version numbers in `run.ver.wcoss2` and use them here? Maybe @MatthewPyle-NOAA knows.

Reply:
Sure. Will do